pkg/storaged/utils.js

```diff
@@ -910,6 +911,15 @@ export function get_active_usage(client, path, top_action, child_action, is_temp
                     enter_unmount(children[c], c, false);
                 enter_unmount(block, mp, true);
             });
+        } else if (swap) {
+            if (swap.Active) {
+                usage.push({
+                    level,
+                    usage: 'swap',
+                    block,
+                    actions: get_actions(_("stop")),
```

Comment:
These 7 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19843-20240116-090759-3ab49c70-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)